dio/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ See the [Migration Guide][] for the complete breaking changes list.**
 ## Unreleased
 
 *None.*
```

Comment:
This can be removed since you added a change

Reply:
thank you, i forgot it